### DISCUSSION TITLE
Improving emission factors for solar  ☀️

### DISCUSSION THREAD

**Discussion by madsnedergaard** [electricitymaps/electricitymaps-contrib#6976] [General]
We'd love to gather the discussions on improving emission factors for solar here.

By the time of this writing, we use IPCC (2014) for the global fallback with a value of 45g CO2eq/kWh.
For Europe, we used [INCER ACV](https://viewer.webservice-energy.org/incer-acv/app/) to calculate the emission factor for each country based on average solar irradiance of the area, giving a range of 25-42g CO2eq/kWh.

Since this production mode is rapidly increasing in capacity across the world, we should investigate if we can find/calculate more up-to-date numbers since efficiency of PV is also increasing.

This is the place to gather and discuss alternative sources, or perhaps consider alternative approaches to calculate more accurate emission factors per country.

**Comment by madsnedergaard** [comment]
As an example, we could consider looking at installation increase per year and use these numbers to calculate the emission factor dynamically for each year.
See data from Energy-Charts here:
https://energy-charts.info/charts/installed_power/chart.htm?l=en&c=DE&expansion=installation_decommission&year=-1&legendItems=cw1&sum=1
https://energy-charts.info/charts/installed_power/chart.htm?l=en&c=EU&expansion=installation_decommission&year=-1&legendItems=jw1&sum=1

**Comment by madsnedergaard** [comment]
Relevant papers to investigate:
- https://www.ise.fraunhofer.de/content/dam/ise/de/documents/publications/conference-paper/wcpec-8/Reichel_5DV234.pdf
- https://www.sciencedirect.com/science/article/abs/pii/S0927024824000369?via%3Dihub

**Comment by Jmuccigr** [comment]
Very late, but glad to see this discussion, especially since the average efficiency of panels has grown since 2014, lowering the cost/KWh.

**Reply by madsnedergaard** [reply]
We haven't been able to do much on this, so if you know of any credible papers on the topic we'd love to hear more :)
